.github/workflows/docs.yml

```diff
@@ -71,15 +71,15 @@ jobs:
           popd
 
       - name: Preview using surge
-        if: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == 'alibaba/GraphAr' && github.repository == 'alibaba/GraphAr' }}
+        if: ${{ github.event_name == 'pull_request_target' && github.event.pull_request.head.repo.full_name == 'alibaba/GraphAr' }}
```

Comment:
Don't use `pull_request_target` as it raises security concern.

If you want to setup preview for docs in PR, try setup something like netlify.

Reply:
See also: https://github.com/alibaba/GraphAr/pull/193